.automation/build-srpm.sh

```diff
@@ -17,10 +17,10 @@ make clean
 # build tarballs
 make dist
 
-SUFFIX=".$(date -u +%Y%m%d%H%M%S).git$(git rev-parse --short HEAD)"
+# SUFFIX=".$(date -u +%Y%m%d%H%M%S).git$(git rev-parse --short HEAD)"
+SUFFIX=''
 
 # Build SRPMs
 rpmbuild \
     -D "_topdir rpmbuild" \
-    -D "release_suffix ${SUFFIX}" \
```

Comment:
Wouldn't it be better to use following?

```
rpmbuild \
    -D "_topdir rpmbuild" \
    ${SUFFIX:- -D "release_suffix ${SUFFIX}"} \
```

Reply:
Yes!

Reply:
I had modified that code because it was not handling correctly empty SUFFIX  ... not sure why.  From the other had I am sure it can be done slightly less verbose compared to my current solution.

Reply:
Ahh, sorry, my mistake, below works fine:
```
rpmbuild \
    -D "_topdir rpmbuild" \
    ${SUFFIX:+ -D "release_suffix ${SUFFIX}"} \
```

Reply:
checks are ok - closing this conversation.